theme-contract/Module.mustache

```diff
@@ -1,32 +1,102 @@
 # Module theme-contract
 
-This is the interface between the library and the custom themes.
 
-#### Tokens versions
+The **OUDS Theme Contract** module is the cornerstone of the design system's theming architecture. It defines the **abstract interfaces** that all concrete themes (Orange, Sosh, Wireframe, etc.) must implement.
+
+It enforces a strict separation of concerns:
+- **The Contract** defines *what* can be styled (e.g., "Brand Primary Color").
+- **The Implementation** defines *how* it looks (e.g., "Orange #FF7900").
+
+## Purpose
+
+This module guarantees consistency across applications. By coding against these interfaces rather than specific implementations, developers can switch themes dynamically without changing a single line of UI code.
+
+## Contract Structure
+
+**Package:** `com.orange.ouds.theme.contract`
+
+This module does not contain any values (hex codes, dp sizes). It only contains:
+1.  **Interfaces:** `OudsThemeContract`, `OudsColorScheme`, `OudsTypography`.
+2.  **Token Objects:** Abstract representations of semantic tokens.
+
+### 1. Color Contract
```

Comment:
Sometimes we have numbers in the headings, sometimes no. I wonder if we should keep them.